examples/nominal.rs

```diff
@@ -0,0 +1,72 @@
+use actix_web::{App, web, Responder,
+  HttpResponse, get, test};
+use biscuit_auth::{Biscuit, KeyPair, 
+  macros::{biscuit, authorizer}, builder_ext::AuthorizerExt};
+use biscuit_actix_middleware::BiscuitMiddleware;
+
+#[get("/hello-admin")]
+async fn hello_admin(token: web::ReqData<Biscuit>) -> impl Responder {
+  // authorizer that allow only if role is admin
+  let mut authorizer = authorizer!(
+    r#"
+      allow if role("admin");
+    "#
+  );
+
+  // deny if no allow
+  authorizer.add_deny_all();
+
+  // link authorizer and token
+  authorizer.add_token(&token)
+    .unwrap();
```

Comment:
question: what happens when a handler panics? Should this error be handled?

Reply:
0 chance to happen, didn't want to add meaningless lines to the example.

But for the beauty, fix in https://github.com/biscuit-auth/biscuit-actix-middleware/commit/415c84cef43bd9ab6dec0b60b82bfc9de34ced76

Reply:
Ah yeah, the authorizer is brand new, so it can't already have a token loaded 👍